docs/BreakingChanges/097 Non-pooled SQL connections will leak memory if not explicitly disposed.md

```diff
@@ -0,0 +1,26 @@
+## 97: Non-pooled SQL connections will leak memory if not explicitly disposed
```

Comment:
{
  "Id": "97",
  "Title": "Non-pooled SQL connections will leak memory if not explicitly disposed",
  "Details": "In the .NET Framework 4.5, non-pooled SQL connections which are not explicitly exposed (via Dispose, Close, or using) will leak memory",
  "Suggestion": "This issue is fixed in a .NET Framework 4.5 servicing update. Please update the .NET Framework 4.5, or upgrade to .NET Framework 4.5.1 or later, to fix this issue. Alternatively, this issue may be avoided by using the SqlConnection in a 'using' pattern (which is a best practice) or by explicitly calling Dispose or Close when the connection is no longer needed.",
  "Link": "https://support.microsoft.com/kb/2748720?wa=wsignin1.0",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Data.SqlClient.SqlConnection.#ctor(System.String)",
    "M:System.Data.SqlClient.SqlConnection.#ctor(System.String,System.Data.SqlClient.SqlCredential)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Data.SqlClient.SqlConnection.#ctor(System.String)
- [x] M:System.Data.SqlClient.SqlConnection.#ctor(System.String,System.Data.SqlClient.SqlCredential)